### PR TITLE
Bug fix develop

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]

File: develop/delete-data.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ TiDB 使用[统计信息](https://docs.pingcap.com/zh/tidb/stable/statistics)来
 
 ### 批量删除例子
 
-假设我们发现在特定时间段内，发生了业务错误，需要删除这期间内的所有 [rating](/develop/bookshop-schema-design.md#ratings-表) 的数据，例如，`2022-04-15 00:00:00` 至 `2022-04-15 00:15:00` 的数据。并且在 15 分钟内，有大于 1 万条数据被写入，我们应该是用循环删除的方式进行删除：
+假设我们发现在特定时间段内，发生了业务错误，需要删除这期间内的所有 [rating](/develop/bookshop-schema-design.md#ratings-表) 的数据，例如，`2022-04-15 00:00:00` 至 `2022-04-15 00:15:00` 的数据。并且在 15 分钟内，有大于 1 万条数据被写入，我们应该使用循环删除的方式进行删除：
```
Comment (qiancai) [inline develop/delete-data.md:127]:
```suggestion
假设发现在特定时间段内，发生了业务错误，需要删除这期间内的所有 [rating](/develop/bookshop-schema-design.md#ratings-表) 的数据，例如，`2022-04-15 00:00:00` 至 `2022-04-15 00:15:00` 的数据。并且在 15 分钟内，有大于 1 万条数据被写入，此时请使用循环删除的方式进行删除：
```

Comment (qiancai) [reply develop/delete-data.md:127]: 尽量避免使用 “我们”

File: develop/choose-driver-or-orm.md [hunk 2]
```diff
@@ -21,7 +21,9 @@ TiDB 与 MySQL 有差异：
 
 ## Java
 
-> 支持等级说明
+> **建议：**
+>
+> 支持等级
 >
 > - Full: 此 Driver 或 ORM 没有已知的 issues
 > - Verified: 你可能会因为 TiDB 兼容性问题，导致出现错误
```
Comment (qiancai) [inline develop/choose-driver-or-orm.md:29]:
```suggestion
建议支持等级：

 - Full: 此 Driver 或 ORM 没有已知的 issues
- Verified: 你可能会因为 TiDB 兼容性问题，导致出现错误
```

Comment (qiancai) [reply develop/choose-driver-or-orm.md:29]: 建议可以不写成注意的形式，用正文即可，不然内容里看起来全是特殊的字体了

Comment (Icemap) [reply develop/choose-driver-or-orm.md:29]: 这是昨天 @Oreoxmt 建议改过来的，原来是非注意格式的 “支持等级说明”。需要改回来嘛？

Comment (Oreoxmt) [inline develop/choose-driver-or-orm.md:29]:
```suggestion
 支持等级：

- Full: 此 Driver 或 ORM 没有已知的 issues
- Verified: 你可能会因为 TiDB 兼容性问题，导致出现错误
```

Comment (qiancai) [reply develop/choose-driver-or-orm.md:29]: 主要是 [docs-cn/choose-driver-or-orm.md at d7fb83b63ca8698115f36ce14ffb96de182fbd24 · pingcap/docs-cn](https://github.com/pingcap/docs-cn/blob/d7fb83b63ca8698115f36ce14ffb96de182fbd24/develop/choose-driver-or-orm.md) 看起来满篇都是建议和注意，感觉阅读体现可能不太好

Comment (Icemap) [reply develop/choose-driver-or-orm.md]: 好的，改回来了。

Comment (Icemap) [reply develop/choose-driver-or-orm.md]: 已改回 **支持等级说明**，去除提醒样式

File: develop/create-secondary-indexes.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ CREATE INDEX `idx_book_published_at` ON `bookshop`.`books` (`bookshop`.`books`.`
 >
 > 执行计划并非每次返回使用的算子都相同，这是由于 TiDB 使用的优化方式为 **基于代价的优化方式 (CBO)**，执行计划不仅与规则相关，还和数据分布相关。你可以前往 [SQL 性能调优](https://docs.pingcap.com/zh/tidb/stable/sql-tuning-overview)文档查看更多 TiDB SQL 性能的描述。
 >
-> TiDB 在查询时，还支持显示的使用索引，你可以使用 [Optimizer Hints](https://docs.pingcap.com/zh/tidb/stable/optimizer-hints) 或 [执行计划管理 (SPM)](https://docs.pingcap.com/zh/tidb/stable/sql-plan-management) 来人为的控制索引的使用。但如果你不了解它内部发生了什么，请你**_暂时先不要使用它_**。
+> TiDB 在查询时，还支持显式的使用索引，你可以使用 [Optimizer Hints](https://docs.pingcap.com/zh/tidb/stable/optimizer-hints) 或 [执行计划管理 (SPM)](https://docs.pingcap.com/zh/tidb/stable/sql-plan-management) 来人为的控制索引的使用。但如果你不了解它内部发生了什么，请你**_暂时先不要使用它_**。
```
Comment (qiancai) [inline develop/create-secondary-indexes.md:145]:
```suggestion
> TiDB 在查询时，还支持显式地使用索引，你可以使用 [Optimizer Hints](https://docs.pingcap.com/zh/tidb/stable/optimizer-hints) 或 [执行计划管理 (SPM)](https://docs.pingcap.com/zh/tidb/stable/sql-plan-management) 来人为的控制索引的使用。但如果你不了解它内部发生了什么，请你**_暂时先不要使用它_**。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**